src/pages/VoucherRedemption/VoucherRedemption.tsx

```diff
@@ -28,6 +29,34 @@ const VoucherRedemption = (props: Props) => {
   const [loading, setLoading] = useState(false);
 
   const fetchData = async () => {
+    let getLocationInfo = (merchantData): LocationInfo => {
+      const emptyLocationInfo = {
+        line1: '',
+        line2: '',
+      };
+      if (!merchantData || !merchantData.data) {
+        return emptyLocationInfo;
+      }
+      const locations = merchantData.data.locations;
+      if (!locations || !locations.length) {
```

Comment:
nit: but I prefer locations.length > 0 to not rely on the truth values of .length

Reply:
Works for me!